odk-build.rst

```diff
@@ -0,0 +1,177 @@
+ODK Build
+===========
+
+:dfn:`ODK Build` is a form designer with a drag-and-drop user interface. Build is an HTML5 web application and works best for designing simple forms.
+
+.. tip::
+
+  For more powerful form designers, try :doc:`XLSForm <xlsform>`, `Vellum <https://github.com/dimagi/Vellum>`_, or `Kobo <http://www.kobotoolbox.org/>`_, or `Enketo <https://enketo.org/>`_ or `PurcForms <https://code.google.com/archive/p/purcforms/>`_.
```

Comment:
For a more powerful form designer from Open Data Kit, try XLSForm.

You may also be interested in form design tools from other organizations:

 - etc...
 - in 
 - a
 - list